lib/crontab/cron_expression/parser.ex

```diff
@@ -193,55 +203,77 @@ defmodule Crontab.CronExpression.Parser do
   defp tokenize(interval, :complex_divider, value) do
     [base, divider] = String.split(value, "/")
 
-    case {tokenize(interval, base), Integer.parse(divider, 10)} do
-      {{:ok, clean_base}, {clean_divider, _}} ->
+    # Range increments apply only to * or ranges in <start>-<end> format
+    range_tokenization_result = tokenize(interval, :-, base)
+    other_tokenization_result = tokenize(interval, base)
+    integer_divider = Integer.parse(divider, 10)
+
+    case {range_tokenization_result, other_tokenization_result, integer_divider} do
+      # Found range in <start>-<end> format
+      {{:ok, clean_base}, _, {clean_divider, _}} ->
         {:ok, {:/, clean_base, clean_divider}}
 
-      {_, :error} ->
-        {:error, "Can't parse " <> value <> " as interval " <> Atom.to_string(interval) <> "."}
+      # Found star (*) range
+      {{:error, _}, {:ok, :*}, {clean_divider, _}} ->
```

Comment:
Here we should probably also match like this:

```suggestion
      {{:error, _}, {:ok, :*}, {clean_divider, ""}} ->
```

Reply:
I'm on it :)

Reply:
I'm adding a clause dedicated to invalid increments in order to provide a more meaningful error message. I'm also adding 2 tests to exercise this.